html/user-list.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>User List</title>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="../css/style.css" />
+</head>
+<body>
+    <nav>
+        <ul>
+            <li><a href="user-profile.html">User Profile</a></li>
+            <li><a href="user-list.html">User List</a></li>
+            <li><a href="create-user.html">Create User</a></li>
+            <li><a href="edit-user.html">Edit User</a></li>
+        </ul>
+     </nav>
+
+    <table>
+        <tr class="cardName">
```

Comment:
My personal preference on class naming is to use `card-name` instead of `cardName`. It helps distinguish between readability between class and id. 

I think "convention" is to use dashes for class-name and camelCase for id, but I'm sure there are arguments the other way.